core/node/vm_runner/Cargo.toml

```diff
@@ -11,17 +11,16 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
-zksync_multivm.workspace = true
+zksync_vm_interface.workspace = true
 zksync_types.workspace = true
 zksync_dal.workspace = true
 zksync_contracts.workspace = true
 zksync_state.workspace = true
 zksync_storage.workspace = true
-zksync_state_keeper.workspace = true
 zksync_utils.workspace = true
 zksync_prover_interface.workspace = true
 zksync_object_store.workspace = true
-zksync_vm_utils.workspace = true
+zksync_vm_executor.workspace = true
```

Comment:
This dependency could be removed similarly to dependency on state keeper provided that batch executor is injected into VM runner components via DI rather than assembled inside components.

Reply:
While it's a good idea, I don't think it's a high priority for now.